client/src/components/ScoreSheet.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Table, Container } from 'semantic-ui-react';
+
+class ScoreSheet extends Component {
+	render() {
+		const { activities, users } = this.props
+		return (
+			<div style={{overflowX: 'scroll'}}>
+				<Table style={{width: '80%', margin: '2% auto'}} celled size='large' definition textAlign='center' unstackable>
```

Comment:
It says it's going to be 80% wide, but it looks 100% on my screen. What is the 80% doing?

![screenshot 2017-10-06 at 9 07 59 am](https://user-images.githubusercontent.com/16890920/31279133-e5625914-aa75-11e7-8956-e78f7d926e8a.png)